workflows/stairway.py

```diff
@@ -57,43 +57,83 @@ def ts_to_stairway(self, ts_path, num_bootstraps=1, mask_file=None):
             total_length += ts.sequence_length
             num_samples = ts.num_samples
             haps = ts.genotype_matrix()
-            # print(haps.shape)
+
+            # Mapping mutation type IDs to class of mutation (e.g., neutral, non-neutral)
+            mut_types = {}
```

Comment:
change the name of `mut_types` here? many things are called "mut_type" around this bit of code